src/App.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import UserProfile from './components/UserProfile';
+import DummyLogin from './components/DummyLogin';
+
+class App extends Component {
+  render () {
+    return (
+      <Router>
+        <Switch>
```

Comment:
Great job @TheDrizzyWay. Please a little explanation on what the `<Switch>` is for?

Reply:
The switch component iterates through every available route, meaning that if the route the user is trying to access does not exist on our application, it would render a 404 component. We would have to build the 404 component ourselves though.

Reply:
Alright...Great job